docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -68,47 +44,86 @@ data:
       namespace: knative-eventing
 ```
 
-Now every Broker created in the cluster that does not have a `spec.config` will be configured to use the `kafka-channel` ConfigMap.
-
-For more information about creating a `kafka-channel` ConfigMap to use with your Broker, see the
-[Kafka Channel ConfigMap](../configuration/kafka-channel-configuration.md#create-a-kafka-channel-configmap) documentation.
-
-### Changing the default Channel implementation for a namespace
-
-You can modify the default Broker creation behavior for one or more namespaces.
+Now every Broker created in the cluster that does not have a `spec.config` will be configured to use the `kafka-channel` ConfigMap. For more information about creating a `kafka-channel` ConfigMap to use with your Broker, see the [Kafka Channel ConfigMap](../configuration/kafka-channel-configuration.md#create-a-kafka-channel-configmap) documentation.
 
-For example, if you wanted to use the `kafka-channel` ConfigMap for all other Brokers created, but wanted to use `config-br-default-channel` ConfigMap for `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
+You can also modify the default Broker configuration for one or more dedicated namespaces, by defining it in the `namespaceDefaults` section. For example, if you want to use the `config-br-default-channel` ConfigMap for all Brokers by default, but want to use `kafka-channel` ConfigMap for `namespace-1` and `namespace-2`, you would use the following ConfigMap:
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: config-br-defaults
   namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
 data:
-  # Configures the default for any Broker that does not specify a spec.config or Broker class.
   default-br-config: |
     clusterDefault:
       brokerClass: MTChannelBasedBroker
       apiVersion: v1
       kind: ConfigMap
-      name: config-kafka-channel
+      name: config-br-default-channel
       namespace: knative-eventing
     namespaceDefaults:
       namespace-1:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
       namespace-2:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
 ```
 
+
+## Configuring the Broker class
+
+Besides configuring the Broker class for each broker [individually](create-broker.md#broker-class-options), it is possible to define the default Broker class cluster wide or on a per namespace basis:
+
+### Configuring the default BrokerClass for the cluster
+
+You can configure the `clusterDefault` Broker class so that any Broker created in the cluster that does not have a `BrokerClass` annotation uses this default class:
```

Comment:
Please use either "BrokerClass" or "Broker class" consistently, depending on how it's used elsewhere in the docs. I think generally we go with Broker class.

Reply:
:+1: Updated. I hope I catched all of them